static/app/gettingStartedDocs/javascript/remix.tsx

```diff
@@ -202,9 +202,20 @@ const feedbackOnboarding: OnboardingConfig = {
           ],
         },
       ],
-      additionalInfo: crashReportCallout({
-        link: 'https://docs.sentry.io/platforms/javascript/guides/remix/user-feedback/#user-feedback-api',
-      }),
+      additionalInfo: (
+        <Fragment>
+          <p>
+            {tct(
+              'Note: The Feedback integration only needs to be added to your [entryClient:entry.client.tsx] file.',
+              {entryClient: <code />, sentryServer: <code />}
```

Comment:
also we no longer need 2x `<code />` thanks to @ArthurKnaus https://github.com/getsentry/sentry/pull/76721 :)

Reply:
Cool, didn't know that was already merged